rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ macro(build_ogre)
 
   if(WIN32)
     set(OGRE_C_FLAGS "${OGRE_C_FLAGS} /w /EHsc")
-    set(OGRE_CXX_FLAGS "${OGRE_CXX_FLAGS} /w /EHsc")
+    set(OGRE_CXX_FLAGS "${OGRE_CXX_FLAGS} -std:c++14 /w /EHsc")
```

Comment:
That's almost certainly not the correct flag to use for MSVC on Windows.  It looks like you'll need something like https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=msvc-170 .

Reply:
-std:c++14 works as well.
I am using Microsoft (R) C/C++ Optimizing Compiler Version 19.29.30133 for x64